Healthy/Classes/Entities/Area.swift

```diff
@@ -0,0 +1,4 @@
+// MARK: - Meal
+struct Area: Codable {
```

Comment:
```suggestion
struct Area {
```

This entity is not meant to be encoded or decoded.